webapp/package-lock.json

```diff
@@ -54,10 +55,120 @@
         "@rollup/rollup-linux-x64-gnu": "^4.21.2"
       }
     },
+    "../../ironrdp/web-client/iron-remote-desktop": {
```

Comment:
issue? Could you re-run npm install and ensure the lock is not referring to local paths?

Reply:
I re-ran it. It should be fine now